.github/ISSUE_TEMPLATE/update_cheatsheet_proposal.md

```diff
@@ -1,16 +1,17 @@
 ---
 name: update_cheatsheet_proposal
-about: Used to allow a proposal to update/refactor a exsiting cheat sheet
+about: Used to create a proposal to update/refactor a exsiting cheat sheet
 title: 'Cheat sheet update/refactor proposal: [PUT_TARGET_CS_NAME_HERE]'
 labels: ACK_WAITING, UPDATE_CS, HELP_WANTED
 assignees: ''
 
 ---
 
-Thanks you for proposing a update/refactor of a cheat sheet.
+Thank you for proposing to update/refactor one of our cheat sheets.
 
-Please provides the following information about your proposal:
+Please provide the following information about your proposal:
 
-- What update/refactoring do you want to perform? 
+1. What is missing or needs to be updated in the current iteration of the cheat sheet?
```

Comment:
Don't you feel like there should be an empty space in between to make even easier for the contributor and for the reviewer to separate the points?

Reply:
Thank you for your feedback 😃 

Just to clarify: do you mean separating the 'missing' or 'needs to be updated' aspects of the question? Or just adding newlines between the points?

Reply:
Newline between the 2 points. Apologies for the confusion, should have been clearer.

Reply:
Sorry for the delay, busy period. I see that you've merged it, thank you 😃

Reply:
Thanks to you for your enhancement of the templates 😃